.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: objective-c
+xcode_workspace: TBODeveloperOverlay.xcworkspace
+xcode_scheme: TBODeveloperOverlay
+xcode_sdk: iphonesimulator
+osx_image: xcode7.2
```

Comment:
Testing on xcode7.3 stalls and results in errors while building. I suspect xctool here.